hack/dockerfiles/generated-files.Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG GO_VERSION="1.20.6"
 ARG PROTOC_VERSION="3.11.4"
 
 # protoc is dynamically linked to glibc so can't use alpine base
-FROM golang:${GO_VERSION}-buster AS base
+FROM golang:${GO_VERSION} AS base
```

Comment:
I think it makes more sense to bump this to `bullseye` instead? Since we do apt commands below, it would be good to keep the debian base clear.

Also, there may be subtle versions between debian versions, that would be good to explicitly opt into.

Reply:
Sure, will use `bookworm` as it's the latest stable.